src/js/lib/content/src/index.html

```diff
@@ -6,10 +6,7 @@
         <!-- FIXME https://github.com/validator/validator/issues/917 -->
         <!-- FIXME Security Vulnerabilities: https://github.com/luna/ide/issues/226 -->
         <meta http-equiv="Content-Security-Policy" content="
-            default-src 'self';
-            script-src  'self' 'unsafe-eval';
-            style-src   'self' 'unsafe-inline';
-            connect-src 'self' data: ws://localhost:* ws://127.0.0.1:*;
```

Comment:
Addressed the issue. Returned to old security policy plus added possibility to load scripts and stylesheets from external https sources, so that everyone can use for example d3.js or mapbox.